browser/ui/webui/brave_webui_source.cc

```diff
@@ -461,19 +461,19 @@ void CustomizeWebUIHTMLSource(const std::string &name,
         { "contributionUpTo",  IDS_BRAVE_REWARDS_LOCAL_CONTR_UP_TO },
 
         { "deviceOffline", IDS_BRAVE_REWARDS_LOCAL_DEVICE_OFFLINE },
-        { "donationTitle",  IDS_BRAVE_REWARDS_LOCAL_DONAT_TITLE },
-        { "donationDesc",  IDS_BRAVE_REWARDS_LOCAL_DONAT_DESC },
-        { "donationTotalDonations",  IDS_BRAVE_REWARDS_LOCAL_DONAT_TOTAL_DONATIONS },            // NOLINT
-        { "donationTotalMonthlyContribution",  IDS_BRAVE_REWARDS_LOCAL_DONAT_TOTAL_MONTHLY_CONTRIBUTION },       // NOLINT
-        { "donationVisitSome",  IDS_BRAVE_REWARDS_LOCAL_DONAT_VISIT_SOME },
-        { "donationAbility",  IDS_BRAVE_REWARDS_LOCAL_DONAT_ABILITY },
-        { "donationAbilityYT",  IDS_BRAVE_REWARDS_LOCAL_DONAT_ABILITY_YT },
-        { "donationAbilityReddit", IDS_BRAVE_REWARDS_LOCAL_DONAT_ABILITY_REDT},
-        { "donationAbilityTwitter",  IDS_BRAVE_REWARDS_LOCAL_DONAT_ABILITY_TW },
-        { "donationAbilityGitHub",  IDS_BRAVE_REWARDS_LOCAL_DONAT_ABILITY_GH },
-        { "donationDisabledText1",  IDS_BRAVE_REWARDS_LOCAL_DONAT_DISABLED_TEXT1 },              // NOLINT
-        { "donationDisabledText2",  IDS_BRAVE_REWARDS_LOCAL_DONAT_DISABLED_TEXT2 },              // NOLINT
-        { "donationNextDate",  IDS_BRAVE_REWARDS_LOCAL_DONAT_NEXT_DATE },
+        { "donationTitle",  IDS_BRAVE_REWARDS_LOCAL_DONATE_TITLE },
+        { "donationDesc",  IDS_BRAVE_REWARDS_LOCAL_DONATE_DESC },
+        { "donationTotalDonations",  IDS_BRAVE_REWARDS_LOCAL_DONATE_TOTAL_DONATIONS },            // NOLINT
+        { "donationTotalMonthlyContribution",  IDS_BRAVE_REWARDS_LOCAL_DONATE_TOTAL_MONTHLY_CONTRIBUTION },       // NOLINT
+        { "donationVisitSome",  IDS_BRAVE_REWARDS_LOCAL_DONATE_VISIT_SOME },
+        { "donationAbility",  IDS_BRAVE_REWARDS_LOCAL_DONATE_ABILITY },
+        { "donationAbilityYT",  IDS_BRAVE_REWARDS_LOCAL_DONATE_ABILITY_YT },
+        { "donationAbilityReddit", IDS_BRAVE_REWARDS_LOCAL_DONATE_ABILITY_REDT},
+        { "donationAbilityTwitter",  IDS_BRAVE_REWARDS_LOCAL_DONATE_ABILITY_TW },
+        { "donationAbilityGitHub",  IDS_BRAVE_REWARDS_LOCAL_DONATE_ABILITY_GH },
+        { "donationDisabledText1",  IDS_BRAVE_REWARDS_LOCAL_DONATE_DISABLED_TEXT1 },              // NOLINT
+        { "donationDisabledText2",  IDS_BRAVE_REWARDS_LOCAL_DONATE_DISABLED_TEXT2 },              // NOLINT
+        { "donationNextDate",  IDS_BRAVE_REWARDS_LOCAL_DONATE_NEXT_DATE },
```

Comment:
I can drop any particular change (they're by replacement string).

This is `DONAT` => `DONATE`. There's no obvious reason to drop one random vowel.